spec/features/user_login_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+feature 'User logs in' do
+  scenario 'as a new user' do
+
+    visit '/'
+
+    fill_in :email, with: "steven@trel.co"
+    fill_in :password, with: "password"
+    click_on "Login"
+
+    expect(current_path).to eq("/address")
```

Comment:
```suggestion
    expect(current_path).to eq("/find")
```
I guess we need to change this test as well, to expect /find.  On this line and line 22